README.md

```diff
@@ -78,6 +88,8 @@ Run `x16emu -h` to see all command line options.
 Keyboard Layout
 ---------------
 
+* ON IPADS A LONG PRESS WILL GIVE YOU ACCESS TO THE FUNCTION KEYS
```

Comment:
I'd rather have a special section that explains what's different on iPad.

Reply:
👍